omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -112,18 +118,25 @@ if [[ -S "/tmp/.s.PGSQL.5432" ]]; then
 fi
 
 # Retrieve Platform and Platform Version
-if [[ -f "/etc/lsb-release" ]] && grep -q DISTRIB_ID /etc/lsb-release; then
-    cp "/etc/lsb-release" "$tmpdir/platform_version.txt"
-    known_platform="ubuntu"
-elif [[ -f "/etc/redhat-release" ]]; then
-    cp "/etc/redhat-release" "$tmpdir/platform_version.txt"
-    known_platform="rhel"
-elif [[ -f "/etc/os-release" ]] && grep -q "Amazon Linux 2" /etc/os-release; then
-    known_platform="amazon"
-    cat "/etc/os-release" >> "$tmpdir/platform_version.txt"
-    cat "/etc/system-release-cpe" >> "$tmpdir/platform_version.txt"
+if [[ -f "/etc/os-release" ]]; then
+    os_name=`grep "^NAME=" /etc/os-release | cut -d'=' -f2- | tr -d '"'`
+    os_version=`grep "^VERSION_ID=" /etc/os-release | cut -d'=' -f2- | tr -d '"'`
+    cp "/etc/os-release" "$tmpdir/platform_os_release.txt"
+    if echo "${os_name}" | grep -iq "SLES"; then
+        known_platform="sles"
+    elif echo "${os_name}" | grep -iq "CentOS"; then
+        known_platform="centos"
+    elif echo "${os_name}" | grep -iq "Red Hat"; then
+        known_platform="rhel"
+    elif echo "${os_name}" | grep -iq "Ubuntu"; then
+        known_platform="ubuntu"
+    elif echo "${os_name}" | grep -iq "Amazon Linux"; then
+        known_platform="amazon"
+    else
+        echo "Platform and version are unknown." > "$tmpdir/platform_os_release.txt"
```

Comment:
These updates look good, but it seems like we're still losing information here.  If we preserve the original copy of os-release that we capture as  platform_os_release.txt instead of overwriting it, it would give support more information than "platform and version unknown".